src/ParallelAlgorithms/LinearSolver/Gmres/Gmres.hpp

```diff
@@ -77,13 +77,25 @@ namespace LinearSolver::gmres {
  * the new orthogonal vector and normalize. Use the residual vector and the set
  * of orthogonal vectors to determine the solution \f$x\f$.
  *
+ * \par Array sections
+ * This linear solver supports running over a subset of the elements in the
+ * array parallel component (see `Parallel::Section`). Set the
+ * `ArraySectionIdTag` template parameter to restrict the solver to elements in
```

Comment:
I'm confused by the statement that they are only in one section based on the template parameter. How does that work with multigrid? I assumed elements were in one section based on their refinement level (and grid index), which is a runtime decision, right?

Reply:
Multigrid creates two sets of sections: One associated with `multigrid::Tags::MultigridLevel` and one with `multigrid::Tags::IsFinestLevel` (note that these tags are not actually necessarily in the DataBox, only the `Parallel::Tags::Section<ParallelComponent, SectionIdTag>` is in the DataBox). The GMRES algorithm runs only on the finest level, and the preconditioners run on all levels.

Reply:
Thanks for explaining that :)